share/availability/light/availability_test.go

```diff
@@ -160,168 +168,3 @@ func BenchmarkService_GetSharesByNamespace(b *testing.B) {
 		})
 	}
 }
-
-func TestSharesRoundTrip(t *testing.T) {
```

Comment:
deleted this test because 
1) its _**only**_ testing app specific things 
2) we have better tests in the app that cover this 
3) this test makes no sense, its loading a pre-written json block and then overwritting the data in it, then using the overwritten data to split into shares, then parsing those shares, then its comparing the inputs and outputs

Reply:
I am ok to remove it. It's a very old test that @tzdybal wrote and if you have better coverage for this in core, lets' drop it